src/PKSim.Core/Services/CKDDiseaseStateImplementation.cs

```diff
@@ -53,23 +57,24 @@ public void ApplyTo(Individual individual)
          var smallIntestinalTransitTime = smallIntestine.Parameter(SMALL_INTESTINAL_TRANSIT_TIME);
          var gastricEmptyingTime = stomach.Parameter(GASTRIC_EMPTYING_TIME);
 
-         var bsaInm2 = bsa.ConvertToUnit(bsa.Value, CoreConstants.Units.m2);
-         var GFR_0 = GFR_spec.Value * kidneyVolume.Value * 1.73 / bsaInm2;
-         if (GFR_0 < targetGFR.Value)
+
+         var GFR_0 = getTargetGFRValue(GFR_spec.Value * kidneyVolume.Value / bsa.Value);
```

Comment:
@Yuri05 You were right. I am creating now a value in l/min/dm2 (no conversion) just based value, and rely on our dimension to give me the value as I need it. Much better